### PR TITLE
Create orgs logic

### PR DESCRIPTION
**Pull request by yvonmanzi**
Added logic to create organizations using [Cryptogen.](https://hyperledger-fabric.readthedocs.io/en/release-2.2/commands/cryptogen.html) The next step is to create organizations using [Fabric CAs](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/users-guide.html#:~:text=The%20Hyperledger%20Fabric%20CA%20is,certificate%20renewal%20and%20revocation) as an alternative to using `Cryptogen `

### CODE REVIEW

File: nalanet/nalanet/organizations/GA/gateway/connection-org1.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          WjBwMQswCQYDVQQGEwJVUzEXMBUGA1UECBMOTm9ydGggQ2Fyb2xpbmExDzANBgNV
+          BAcTBkR1cmhhbTEZMBcGA1UEChMQb3JnMS5leGFtcGxlLmNvbTEcMBoGA1UEAxMT
+          Y2Eub3JnMS5leGFtcGxlLmNvbTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABF86
+          cUpFpQTWImevPjSO+OXypA0Y0BiWtxiFynYH/Ap8lutL0qnzrlHU62bYIRN8oI9w
+          DIpWpovFve02PF7lnmmjRTBDMA4GA1UdDwEB/wQEAwIBBjASBgNVHRMBAf8ECDAG
+          AQH/AgEBMB0GA1UdDgQWBBTAwLBJICVMYlm4NNYHJWoxGZmKFjAKBggqhkjOPQQD
+          AgNHADBEAiAB/SMG6j8GO7NcC4R4qVBE9VtRcC5qLjWSMum9i2LvCAIgdqTVKElQ
+          AJRUwvdDj/ZnNMQS+uGVchta864R4cEGwCk=
+          -----END CERTIFICATE-----
+
```
Comment (YuanSamuel) [inline nalanet/nalanet/organizations/GA/gateway/connection-org1.yaml:36]: Will we have to remove/hide these once we deploy?

Comment (yvonmanzi) [reply nalanet/nalanet/organizations/GA/gateway/connection-org1.yaml:36]: yea that's just a dummy thing I was trying.  i thought I had already removed that. thanks for pointing it out. i will go ahead and remove it now.

Comment (yvonmanzi) [reply nalanet/nalanet/organizations/GA/gateway/connection-org1.yaml:36]: wait, is there a way i could make another PR off of this?

Comment (yvonmanzi) [reply nalanet/nalanet/organizations/GA/gateway/connection-org1.yaml:36]: Actually, let's not merge this PR now. it needs a lot more work. i only did half of the entire thing.

**Pull request closed without merging**